docs/hyproof/api-flow.md

```diff
@@ -49,232 +47,258 @@ The values set for each persona are your choice but, they should provide a recog
 
 ```sh
-curl -s http://localhost:9000/v1/self -H 'accept: application/json' | jq -r .address
-# 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
-heidi=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
-curl -s http://localhost:9010/v1/self -H 'accept: application/json' | jq -r .address
-# 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
-emma=5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
-curl -s http://localhost:9020/v1/self -H 'accept: application/json' | jq -r .address
-# 5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y
-reginald=5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y
+heidi=$(curl -s http://localhost:9000/v1/self -H 'accept: application/json' | jq -r .address)
+emma=$(curl -s http://localhost:9010/v1/self -H 'accept: application/json' | jq -r .address)
+reginald=$(curl -s http://localhost:9020/v1/self -H 'accept: application/json' | jq -r .address)
 ```
 
-2. Set the alias for all the personas using the 1st Swagger w/ **`POST`** **`/members/{address}`**:
+These commands should populate the following environment variables:
+```sh
+heidi = 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+emma = 5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty
+reginald = 5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y
+```
+
+2. Set the aliases for all the personas using the 1st Swagger w/ **`POST`** **`/members/{address}`**:
 
 ```sh
-curl -X 'PUT' http://localhost:9000/v1/members/${heidi} \
+curl -X 'PUT' http://localhost:9000/v1/members/$heidi \
   -H 'Content-Type: application/json' \
-  -d '{"alias":"heidi"}'
+  -d '{"alias":"Heidi"}'
 
-curl -X 'PUT' http://localhost:9000/v1/members/${emma} \
+curl -X 'PUT' http://localhost:9000/v1/members/$emma \
   -H 'Content-Type: application/json' \
-  -d '{"alias":"emma"}'
+  -d '{"alias": "Emma"}'
 
-curl -X 'PUT' http://localhost:9000/v1/members/${reginald} \
+curl -X 'PUT' http://localhost:9000/v1/members/$reginald \
   -H 'Content-Type: application/json' \
-  -d '{"alias": "reginald"}'
-```
-
-```js
-// Payloads:
-'{"alias":"heidi"}'
-'{"alias":"emma"}'
-'{"alias": "reginald"}'
+  -d '{"alias": "Reginald"}'
 ```
 
-3. Set the alias for all the personas using the 2nd Swagger w/ **`POST`** **`/members/{address}`**:
+3. Set the aliases for all the personas using the 2nd Swagger w/ **`POST`** **`/members/{address}`**:
 
 ```sh
-curl -X 'PUT' http://localhost:9010/v1/members/${heidi} \
+curl -X 'PUT' http://localhost:9010/v1/members/$heidi \
   -H 'Content-Type: application/json' \
-  -d '{"alias":"heidi"}'
+  -d '{"alias":"Heidi"}'
 
-curl -X 'PUT' http://localhost:9010/v1/members/${emma} \
+curl -X 'PUT' http://localhost:9010/v1/members/$emma \
   -H 'Content-Type: application/json' \
-  -d '{"alias":"emma"}'
+  -d '{"alias": "Emma"}'
 
-curl -X 'PUT' http://localhost:9010/v1/members/${reginald} \
+curl -X 'PUT' http://localhost:9010/v1/members/$reginald \
   -H 'Content-Type: application/json' \
-  -d '{"alias": "reginald"}'
+  -d '{"alias": "Reginald"}'
 ```
```

Comment:
not sure if it works, but just remembered
```sh
for persona in $heidi $emma $reginald
do
  curl -X 'PUT' http://localhost:9010/v1/members/$persona -H 'Content-Type: application/json' -d '{"alias": $persona }'
done
```

Reply:
This would make life a lot easier. Let me test.